src/azure-cli/HISTORY.rst

```diff
@@ -51,6 +51,10 @@ Release History
 * Upgrading SDK Version to 0.6.0
 * Add billingplan details info after calling Get-Gatalogs
 
+**SQL**
+
+* Update azure-mgmt-sql to version 0.15.0 which contains changed signature for operations get/delete/create_or_update in managed_instance_administrators.
```

Comment:
This is an internal change so I think it doesn't need to be announced here.

Reply:
It is removed now.

Reply:
@jaredmoo @SanjaMalesevic Hello. User should know the change of dependency. We can keep "Update azure-mgmt-sql to version 0.15.0" without "which contains changed signature for operations get/delete/create_or_update in managed_instance_administrators."